api/payment/stripeRouter

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const router = express.Router();
+const stripe = require('stripe')(process.env.STRIPE_SECRET_KEY);
```

Comment:
Have we added this `STRIPE_SECRET_KEY` to any of our documentation? Want to make sure this works on all of our locals, and it'll eventually need to be added to deployment as well!